CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 3.0.0
+
+- Add retry logic.
+
+** Possible Breaking Change ** Error messages may now be delayed up to 5 seconds
```

Comment:
Should we say "Error messages for disconnects"?

Reply:
General error messages may be delayed as we have no way to make a distinction.